frontend/src/e2e-test/specs/5_employee/realtime-attendances.spec.ts

```diff
@@ -0,0 +1,143 @@
+} from '../../dev-api/fixtures'
+import { Child, Daycare, EmployeeDetail } from '../../dev-api/types'
+import { UnitPage } from '../../pages/employee/units/unit'
+import { UnitAttendancesPage } from '../../pages/employee/units/unit-attendances-page'
+import { Page } from '../../utils/page'
+import { employeeLogin } from '../../utils/user'
+
+let page: Page
+let unitPage: UnitPage
+let calendarPage: UnitAttendancesPage
+let child1Fixture: Child
+let child1DaycarePlacementId: UUID
+let daycare: Daycare
+let unitSupervisor: EmployeeDetail
+let staff: EmployeeDetail[]
+
+const mockedToday = LocalDate.of(2022, 3, 28)
+const placementStartDate = mockedToday.subWeeks(4)
+const placementEndDate = mockedToday.addWeeks(4)
+const groupId: UUID = uuidv4()
+
+beforeEach(async () => {
+  await resetDatabase()
+
+  const fixtures = await initializeAreaAndPersonData()
+  const careArea = await Fixture.careArea().with(careArea2Fixture).save()
+  await Fixture.daycare().with(daycare2Fixture).careArea(careArea).save()
+
+  daycare = daycare2Fixture
+
+  unitSupervisor = (await Fixture.employeeUnitSupervisor(daycare.id).save())
+    .data
+
+  await insertDefaultServiceNeedOptions()
+
+  await Fixture.daycareGroup()
+    .with({
+      id: groupId,
+      daycareId: daycare.id,
+      name: 'Testailijat'
+    })
+    .save()
+
+  child1Fixture = fixtures.familyWithTwoGuardians.children[0]
+  child1DaycarePlacementId = uuidv4()
+  await Fixture.placement()
+    .with({
+      id: child1DaycarePlacementId,
+      childId: child1Fixture.id,
+      unitId: daycare.id,
+      startDate: placementStartDate.formatIso(),
+      endDate: placementEndDate.formatIso()
+    })
+    .save()
+
+  await Fixture.groupPlacement()
+    .with({
+      daycareGroupId: groupId,
+      daycarePlacementId: child1DaycarePlacementId,
+      startDate: placementStartDate.formatIso(),
+      endDate: placementEndDate.formatIso()
+    })
+    .save()
+
+  staff = [
+    (await Fixture.employeeStaff(daycare.id).save()).data,
+    (
+      await Fixture.employee()
+        .with({
+          email: 'kalle.kasvattaja@evaka.test',
+          firstName: 'Kalle',
+          lastName: 'Kasvattaja',
+          roles: []
+        })
+        .withDaycareAcl(daycare.id, 'STAFF')
+        .save()
+    ).data
+  ]
+  await Fixture.staffOccupancyCoefficient(daycare.id, staff[1].id).save()
+
+  page = await Page.open({
+    mockedTime: mockedToday.toSystemTzDate(),
+    employeeCustomizations: {
+      featureFlags: {
+        citizenShiftCareAbsenceEnabled: true,
+        assistanceActionOtherEnabled: true,
+        daycareApplication: {
+          dailyTimesEnabled: true
+        },
+        groupsTableServiceNeedsEnabled: false,
+        evakaLogin: true,
+        financeBasicsPage: true,
+        preschoolEnabled: true,
+        urgencyAttachmentsEnabled: true,
+        adminSettingsEnabled: false,
+        experimental: {
+          realtimeStaffAttendance: true
+        }
+      }
+    }
+  })
```

Comment:
You don't have to specify all feature flags if you take the defaults and override what's needed:

```
import { featureFlags } from 'lib-customizations/employee'

...

  page = await Page.open({
    mockedTime: mockedToday.toSystemTzDate(),
    employeeCustomizations: {
      featureFlags: {
        ...featureFlags,
        experimental: {
          realtimeStaffAttendance: true
        }
      }
    }
  })
```

Reply:
I could have sworn I tried to do this but my env refused to find the import. It does work now, though 🤔